docs/modules/ROOT/pages/getting-started/filters-and-projections.adoc

```diff
@@ -0,0 +1,191 @@
+= Filters and projections
+
+After making a basic `MATCH` & `RETURN` Cypher query in xref:getting-started/your-first-query.adoc[Your First Query], in this guide you'll add some filters and change the projection to only return some fields.
+
+The resulting query will look like this:
+
+
+```cypher
+MATCH(m:Movie)
+WHERE (m.title = "The Matrix" AND m.released < 2000)
+RETURN m.title, m.tagline, m.released
+```
+
+NOTE: This query will return all movies with the name `"The Matrix"` released before `2000`. This query is based on the default example database in Neo4j, and will return the classic 1999 film **The Matrix**. 
+
+
+== Filtering
+
+When creating a `MATCH` clause, it is possible to add subclauses to it in a similar way to how you added the `RETURN` statement. Change the `Cypher.Match` instance to add a `WHERE` subclause with the method `.where`:
+
+```javascript
+const clause = new Cypher.Match(movieNode)
+    .where(Cypher.eq(movieNode.property("title"), new Cypher.Param("The Matrix")))
+    .return(movieNode);
+```
+
+There is quite a bit to unwrap here. First, you'll notice that the `.where` method takes the result of the function `Cypher.eq`. This function is the **eq**uality (`=`) operator that compares the property `title` and the string `The Matrix`.
+
+`Cypher.eq` takes two arguments to compare. Our first argument will be the property `title`. Because our title is a property of our movieNode variable, the `.property` method of that node will ensure that the property correctly references the movieNode variable.
+
+The second parameter is `"The Matrix"`, but instead of using a plain string, the object `Cypher.Param` class will take care of turning that into a parameter so we can safely put user input here.
+
+
+.Why `Cypher.eq` doesn't need `new`?
+[NOTE]
+====
+Cypher Builder treats Variables (like nodes and relationships) and Clauses as classes, as these are intended to be assigned to a JavaScript variable to be reused, composed and built.
```

Comment:
```suggestion
Similarly to nodes and relationships, Cypher Builder treats variables and clauses as classes.
This is because they are intended to be reused, composed, and built.
```